hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/DelimitedKeyPrefixRegionSplitPolicy.java

```diff
@@ -37,7 +37,11 @@
  * <code>userid_eventtype_eventid</code>, and use prefix delimiter _, this split policy
  * ensures that all rows starting with the same userid, belongs to the same region.
  * @see KeyPrefixRegionSplitPolicy
+ *
+ * @deprecated since 2.4.3 and will be removed in 4.0.0. Use {@link RegionSplitRestriction},
```

Comment:
You need to update this deprecation string to also include the applicable 2.3.x version number.